examples/offline/offline_bcq.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python3
+import argparse
+import datetime
+import os
+import pprint
+
+import d4rl
+import gym
+import numpy as np
+import torch
+from torch.utils.tensorboard import SummaryWriter
+
+from tianshou.data import Batch, Collector, ReplayBuffer, VectorReplayBuffer
+from tianshou.env import SubprocVectorEnv
+from tianshou.policy import BCQPolicy
+from tianshou.policy.imitation.bcq import VAE, Perturbation
```

Comment:
Do you think moving `VAE` and `Pertubation` to `utils/net/...` is a good idea or not? Because previously we all place our network architecture under that module.

Reply:
BTW it's easier for other algorithms to reuse VAE, if needed.

Reply:
Yes. I think VAE may be useful for other algorithms.